app/models/access_control_entry.rb

```diff
@@ -1,4 +1,10 @@
 class AccessControlEntry < ApplicationRecord
   acts_as_tenant(:tenant)
   belongs_to :aceable, :polymorphic => true
+  has_many :access_control_permissions, :dependent => :destroy
+  has_many :permissions, :through => :access_control_permissions
+
+  def add_permissions(permissions)
+    permissions.each { |name| self.permissions << Permission.find_by!(:name => name) }
```

Comment:
@syncrou Shouldn't these be checking for duplicates, if the object already has a read permission will this keep adding rows to the join table.

add_permissions(['read','read','read'])
&
add_permission('read')
add_permission('read')
add_permission('read')